avocado_vt/loader.py

```diff
@@ -79,6 +79,28 @@ def arch_listing(config, guest_name_parser=None):
     LOG.debug("")
 
 
+def get_vt_test_tags(vt_params):
+    """
+    Returns the test categories based on a `tags` vt param
+
+    :rtype: dict
```

Comment:
This could benefit from a complete docstring like:
```
Get the test categories based on a `tags` VT parameter.

:param vt_params: currently parsed VT parameters
:type vt_params: dict
:returns: test categories based on a `tags` vt param
:type: dict
```

Reply:
Ok, thanks for your suggestion, will update in the next version.